src/scripts/comment-pr.sh

```diff
@@ -38,6 +38,12 @@ function isAuthenticatedGitHub() {
 }
 
 function mainGitHub() {
+  if [ "${CIRCLE_PULL_REQUEST}" == "" ]; then
```

Comment:
```suggestion
  if [ -z "${CIRCLE_PULL_REQUEST}" ]; then
```
Suggesting swapping over to a bash built-in for detecting the value of the env var

Reply:
To be completely honest I am not _confident_ this will solve the issue but it's very hard to see without trying. I do not believe this will cause any issues though so let's move forward 👍